api_spec/edge_api.json

```diff
@@ -0,0 +1,205 @@
+                        "application/json": {
+                            "schema": {
+                                "type": "object",
+                                "items": {
+                                    "$ref": "#/components/schemas/createBodyRequest"
+                                }
+                            }
+                        }
+                    }
+                },
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "array",
+                                    "items": {
+                                        "$ref": "#/components/schemas/Flag"
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "components": {
+        "securitySchemes": {
+            "EnvironmentKeyAuth": { "type": "apiKey", "in": "header", "name": "X-Environment-Key" }
+        },
+        "schemas": {
+            "Flag": {
+                "required": [],
+                "type": "object",
+                "properties": {
+                    "feature": {
+                        "$ref": "#/components/schemas/Feature"
+                    },
+                    "feature_state_value": {
+                        "$ref": "#/components/schemas/FeatureStateValue"
+                    },
+                    "enabled": {
+                        "title": "Enabled",
+                        "type": "boolean"
+                    }
+                }
+            },
+            "FeatureStateValue": {
+                "type": "object",
+                "description": "Can be any of the following types: string, integer, boolean, float"
+            },
+            "Feature": {
+                "title": "Feature",
+                "required": ["name"],
+                "type": "object",
+                "properties": {
+                    "id": {
+                        "title": "ID",
+                        "type": "integer",
+                        "readOnly": true
+                    },
+                    "name": {
+                        "title": "Name",
+                        "type": "string",
+                        "maxLength": 2000,
+                        "minLength": 1
+                    },
+                    "type": {
+                        "title": "Type",
+                        "type": "string",
+                        "maxLength": 50
+                    }
+                }
+            },
+            "createBodyRequest": {
+                "title": "object",
+                "required": true,
+                "type": "object",
+                "properties": {
+                    "Identifier": {
```

Comment:
I think you've got the object key / title attribute the wrong way around. The object key should be the attribute that we expect the user to provide. The title is a user friendly name for the attribute. For example, this should be: 

```
"identifier": {
    "title": "Identifier"
    ...
}
```

Similarly for the traits, we should have something like: 

```json
"traits": {
    "title": "Traits"
    ...
}
```